javascript/javascript3/week3/art.js

```diff
@@ -0,0 +1,45 @@
+class Circle {
+    constructor(x, y, r, startAngle, endAngle, fillColor) {
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngle = startAngle;
+        this.endAngle = endAngle;
+        this.fillColor = fillColor;
+    }
+    draw(){
+        var canvas = document.querySelector("canvas");
+        var ctx = canvas.getContext("2d");
+        ctx.beginPath();
+        ctx.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
+        ctx.fillstyle = this.fillColor;
```

Comment:
The color of the circles are not changing. That's because there is a typo in line 15. Use `fillStyle` instead of `fillstyle`
```suggestion
        ctx.fillStyle = this.fillColor;
```